haskell/private/actions/compile.bzl

```diff
@@ -322,9 +329,7 @@ def _compilation_defaults(hs, cc, java, dep_info, plugin_dep_info, cc_info, srcs
         args = args,
         compile_flags = compile_flags,
         inputs = depset(transitive = [
-            depset(header_files),
-            depset(boot_files),
-            set.to_depset(source_files),
+            source_files,
```

Comment:
Why are `boot_files` and `header_files` no longer included here?

Reply:
Because they're part of extra_source_files.